pmd-modelica/src/main/resources/category/modelica/bestpractices.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0"?>
+
+<ruleset name="Best Practices"
+         xmlns="http://pmd.sourceforge.net/ruleset/2.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://pmd.sourceforge.net/ruleset/2.0.0 https://pmd.sourceforge.io/ruleset_2_0_0.xsd">
+    <description>
+Rules which enforce generally accepted best practices.
+    </description>
+
+    <rule name="ClassStartNameEqualsEndName"
+          language="modelica"
+          message="Class ends with an end clause with a different name"
+          class="net.sourceforge.pmd.lang.rule.XPathRule">
```

Comment:
I think, the externalInfoUrl attribute is missing - it should be:

`externalInfoUrl="${pmd.website.baseurl}/pmd_rules_modelica_bestpractices.html#classstartnameequalsendname"`

The same applies for the other two rules here.
Note: the rule documentation will be generated and is not published yet